src/50_Samples_%26_Templates/Product.html

```diff
@@ -124,36 +131,48 @@
     <body>
     <%>product/header.html%>
 
-        <!-- #### Product Quantity -->
-        <!-- This dropdown is realized using [amp-form](/components/amp-form/) and allows to select how many items to be added to the shopping cart. Pressing the `ADD TO CART` button leads to a shopping cart page page. -->
-        <div id="product-header">
-            <h2>Apple</h2>
+    <h2>Apple</h2>
+    <p id="product-description">Lorem ipsum dolor sit amet, ut sed non vel mattis. Et nulla suscipit ante ligula nisl in. Ut venenatis et mauris, mauris porta vulputate tellus mauris integer facilisis.</p>
+    <h5 class="price-description">From $1.99</h5>
+
+     <!-- #### Image Gallery -->
+    <!-- The `amp-carousel` works very well for product image galleries. Learn more about `amp-carousel` [here](/advanced/image_galleries_with_amp-carousel/). -->
+    <amp-carousel width="512" height="384" layout="responsive" type="slides">
+        <amp-img src="/img/product1_1024x682.jpg" width="1024" height="682" layout="responsive"></amp-img>
+        <amp-img src="/img/product1_alt1_1024x682.jpg" width="1024" height="682" layout="responsive"></amp-img>
+        <amp-img src="/img/product1_alt2_1024x682.jpg" width="1024" height="682" layout="responsive"></amp-img>
+    </amp-carousel>
+
+
+    <!-- #### Product Quantity -->
+    <!-- This dropdown is realized using [amp-form](/components/amp-form/) and allows to select how many items to be added to the shopping cart. Pressing the `ADD TO CART` button leads to a shopping cart page. We are using the `CLIENT_ID` variable to identify the user. This variable can be used inside an amp-form by declaring an input hidden value with `default-value="CLIENT_ID(cart)`.  In this sample, the server is using it to map the user to a shopping cart. Read more about variable substitution [here](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md)-->
```

Comment:
* ' Pressing the `ADD TO CART` button leads to a shopping cart page. ' is no longer correct
* Could you maybe elaborate a bit more what 'identify the user' means? I think it is important to explain here why we need to identify the user and what's the problem we're solving. 
* s/input hidden value/hidden input value/
* Can you explain 'In this sample, the server is using it to map the user to a shopping cart.'?

*